README.md

```diff
@@ -46,7 +46,7 @@ How about:
 ![tldr screenshot](images/screenshot.png)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX / Linux / macOS / SunOS commands.
+for the most common UNIX, Linux, OS X, SunOS and Windows commands.
```

Comment:
This is not quite correct. The new name for OS X is macOS - this should be changed.

Reply:
Hmm, that's true. I must have been out of date with this, as this has changed back in 2016. :confused: